toolchain-m68k

```diff
@@ -241,6 +241,14 @@ def build():
                               path.join('{host}', 'bin'),
                               environ['PATH']])
 
+  """
+  When we have a working compiler in our path, we can also check if the
+  required headers/libraries are present.
+  """
+
+  require_header('<python2.7/python.h>')
```

Comment:
I mentioned `python-dev` in prerequisites, but I don't remember if it's actually required. I cannot find any traces of `python.h` use O.o